### ISSUE TITLE
Unknown software license

### ISSUE THREAD

**Issue by andrewreutter** [seethroughtrees/react-wufoo-embed#6]
Hi,

Is there a license associated with this library? I note the copyright claim, but there's no LICENSE file in the package. I'd like to use it on a work project, but our policies prevent the use of libraries that don't use a license on our whitelist (which is fairly lengthy, we're happy with MIT, GPL, Apache, ISC, etc...). 

Thanks in advance!

**Comment by seethroughtrees** [comment]
Hi @andrewreutter .  Thanks for pointing that out.  I just added the GPL license.  Let me know if that works for you.  I'm happy to switch it to any other if it helps.

Also, I haven't used or updated this in a few years, so it might be worth looking over before you use it in your project.

Hope it helps.

**Comment by andrewreutter** [comment]
Wow, what an amazingly quick and friendly response, thanks.

I got a clarification about our GPL license policy, and it turns out that
it's allowed on the back end but not for anything executed in the browser,
because that triggers the distribution clauses and force us to open-source
software that we'd rather not.

I apologize for the earlier miscommunication. Any chance you're willing to
go with the MIT license? I think most npm packages go with the MIT license
for the very reasons I described above.

Also, thanks for being up-front about the maintenance status of the project.

- Andrew
